docs/docs/guides/migration_from_other_libs/ethers.md

```diff
@@ -0,0 +1,478 @@
+---
+sidebar_position: 1
+sidebar_label: 'Migration from ethers.js'
+title: 'Migration from ethers.js'
+---
+
+Follow this guide, if you're currently using the ethers.js library to interact with the Ethereum blockchain and want to migrate to web3.js.
```

Comment:
Is this guide for ether.js v5 to web3.js? I think it should be mentioned at start.

Reply:
It is for both, where if the code was different between 5 and 6. then I provided the different code. I will add this to the file.